libs/coin-modules/coin-aptos/src/bridge/broadcast.ts

```diff
@@ -1,16 +1,19 @@
 import type { Account, Operation, SignedOperation } from "@ledgerhq/types-live";
 import { patchOperationWithHash } from "@ledgerhq/coin-framework/operation";
-import { AptosAPI } from "../api";
+import { createApi } from "../api";
```

Comment:
why not use `import { AptosAPI } from "../network";` and keep the same approach? I wouldn't use the `createApi` in the bridge